qutebrowser/browser/commands.py

```diff
@@ -2167,9 +2171,13 @@ def _open_if_changed(self, url=None, old_url=None, bg=False, tab=False,
             bg: Open in a new background tab.
             tab: Open in a new tab.
             window: Open in a new window.
+            private: Open a new window in private browsing mode.
+            related: If opening a new tab, position the tab as related to the
+                     current one (like clicking on a link).
         """
-        if bg or tab or window or url != old_url:
-            self.openurl(url=url, bg=bg, tab=tab, window=window)
+        if bg or tab or window or private or related or url != old_url:
```

Comment:
pylint complains with `too-many-boolean-expressions` here, but I'm inclined to just turn that check off. What do you think?

Reply:
agreed